L1Trigger/DTTriggerPhase2/src/MPThetaMatching.cc

```diff
@@ -0,0 +1,325 @@
+#include "L1Trigger/DTTriggerPhase2/interface/MPThetaMatching.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "L1Trigger/DTTriggerPhase2/interface/constants.h"
+
+using namespace edm;
```

Comment:
Is it possible to avoid the `using namespace edm` please?

Reply:
Sure, sorry